### PR TITLE
Add more stayman followups

### PR DESCRIPTION
**Pull request by arigreen**
I'm not sure I see the value of using 3C to show either minor and 3D to confirm a M fit.

It seems the pros are:
- More choice of game auctions
- More room to cuebid for slam auctions

And the cons are:
 - Less room for exploring when responder has a 3m bid.

It would be nice to bid some hands here.

### CODE REVIEW

File: 1nt.tex [hunk 1]
```diff
@@ -52,4 +54,20 @@ \chapter{1NT}
...
+\orauction{1n,2c,2M,?}
+\begin{description}
+  \item[\sp2] (Over \he2). 5\spadesuit, invitational. Same followups as over 1NT-\cl2-\di2-\sp2.
+  \item[2NT] Invitiational. Promises 4 cards in OM.
+  \item[\cl3] 5+ \clubsuit ~OR 5+ \diamondsuit, GF. \di3 asks for the minor, LH. Other bids are natural, including bidding the other major to confirm a fit there.
+  \item[\di3] Artificial, confirms a fit in M, typically no shortness. Opener can bid 3M to suggest playing in M, 3NT to suggest playing there, or 4M to insist on M, denying slam interest. Other suit bids are cuebids with hands well suited for slam.
+  \item[3M] Invitational
+  \item[3OM] Unspecified splinter. Next step asks, LMH.
+  \item[3NT] To play
+  \item[\cl4] 4M, 6OM, slam try. Opener's 4OM rebid is an offer to play.
```
Comment (arigreen) [inline 1nt.tex]: Do you want to play this? Also, do you agree that 4OM is an offer to play here. Should double keycard apply here as well?

File: 1nt.tex [hunk 2]
```diff
@@ -52,4 +54,20 @@ \chapter{1NT}
   \item[3NT] Probably a 2-card maximum that forgot to bid \cl3.
 \end{description}
 
+\orauction{1n,2c,2M,?}
+\begin{description}
+  \item[\sp2] (Over \he2). 5\spadesuit, invitational. Same followups as over 1NT-\cl2-\di2-\sp2.
+  \item[2NT] Invitiational. Promises 4 cards in OM.
+  \item[\cl3] 5+ \clubsuit ~OR 5+ \diamondsuit, GF. \di3 asks for the minor, LH. Other bids are natural, including bidding the other major to confirm a fit there.
```
Comment (arigreen) [inline 1nt.tex]: I'm not clear if 3C bidder an have 4M and not 4OM. If so, how do the followups resolve this?

Comment (tomc) [reply 1nt.tex]: In theory it should be 4OM 100% of the time, but the wiggle room there is a nod to Joel back in the day who didn't like that.  I'm fine with explicitly stating that it shows the OM.

**Comment by tomc** [comment]
I can't seem to reply to the 3C/3D comment, so posting down here.

I think there is great value in having a GF call agreeing the 4 card suit below game, but I'm fine with bidding some hands.  I'll ponder a bit and see if I can come up with a better solution than overloading the 3C bid.  Off the cuff, playing 3C as the GF and 3D/3OM as C/D respectively might work - we'd need to incorporate the splinter hands into the 3C bid then, but that's feasible.

File: 1nt.tex (hunk 2 above)
Comment (arigreen) [reply 1nt.tex]: I could imagine a 4M 6m hand with support for opener's suit that wants to show the minor to better assess slam potential. Something like x.Axxx.Kx.AJTxxx where how good the slam is is dependent on opener's fit for the minor. If you like, we could utilize additional steps over 3D to show this:
1N-2C-2H-3C-3D
3H = 4xx5
3S = 4x5x
3N = x4x5
4C = x45x

In all examples above, 5 is really 5+.

**Pull request merged**